velox/docs/functions/presto/regexp.rst

```diff
@@ -18,6 +18,11 @@ See https://github.com/google/re2/wiki/Syntax for more information.
     Note: The wildcard '%' represents 0, 1 or multiple characters and the
     wildcard '_' represents exactly one character.
 
+    Note: Each function instance allow for a maximum of 20 regular expressions to
```

Comment:
Since we usually run 15 threads, this give us 20 * 15 = 300 regex per function call per query. Should be enough. If not, let's look into specific use cases to understand why they need more. CC: @spershin @pranjalssh

Reply:
It is still more like 20 as data might be split any way among different threads. We need to dig into some of the queries and see how much they need, and if this approach will work. Have we done that @laithsakka ?